### PR TITLE
doc: reorg attestation/signing docs

### PR DESCRIPTION
**Pull request by Gankra**

### CODE REVIEW

File: book/src/supplychain-security/signing-windows.md [hunk 1]
```diff
@@ -57,15 +55,15 @@ Want support for another vendor? [Drop us a line](mailto:hello@axo.dev) or [file
...
+### Part 2: Enable SSL.com signing with cargo-dist
 
 1. **Configure cargo-dist to codesign**
 
     Add the following to your `Cargo.toml` or `dist.toml`:
 
     ```toml
     [workspace.metadata.dist]
-    ssldotcom-windows-sign = "prod" # or "test" if you are using a sandbox account
+    ssldotcom-windows-sign = "prod"  or "test" if you are using a sandbox account
```
Comment (ashleygwilliams) [inline book/src/supplychain-security/signing-windows.md:66]: @Gankra is this on purpose?

Comment (Gankra) [reply book/src/supplychain-security/signing-windows.md:66]: Sure wasn't

Comment (Gankra) [reply book/src/supplychain-security/signing-windows.md:66]: oh lmao i did `s/# /` to lower the level of each heading by one

File: book/src/supplychain-security/attestations/github.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 GitHub's [Artifact Attestations] feature - currently in public beta - allows for the creation of a tamper-proof, unforgeable paper trail linking build artifacts to the process which created it. Artifact Attestations is powered by [Sigstore], an open source project for signing and verifying software artifacts.
 
-Artifact Attestations is disabled by default in `cargo-dist`, and can be enabled by setting the `github-attestions` config to `true`.
+Artifact Attestations is disabled by default in `cargo-dist`, and can be enabled by [setting `github-attestations = true`](../reference/config.md#github-attestations)
```
Comment (Gankra) [inline book/src/supplychain-security/attestations/github.md:5]:
```suggestion
Artifact Attestations is disabled by default in `cargo-dist`, and can be enabled by [setting `github-attestations = true`](../../reference/config.md#github-attestations)
```

**Review by ashleygwilliams — APPROVED** [review]

**Pull request merged**